CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+## Process
+
+We use a fork-and-PR process, also known as a triangular workflow. This process
+is fairly common in open-source projects. Here's the basic workflow:
+
+1. *Fork* the upstream repo to create your own repo. This repo is called the origin repo.
+2. *Clone* the origin repo to create a working directory on your local machine. 
```

Comment:
@esatterwhite, why are these verbs in `1.` and `2.` in `italic` but not others?

Reply:
Errrr that was me being picky. Trying to, well, emphasize the two words. ;-) Feel free to remove the asterisks and free the verbs if you like.

Reply:
removed